script.js

```diff
@@ -243,3 +243,45 @@ ${dynamicImage}
     }
   });
 }
+
+// Javascript for the local storage
+const userDataInLocalStorage = localStorage.getItem('userData');
+const parsedData = JSON.parse(userDataInLocalStorage);
+document.getElementById('fullName').value = parsedData.name;
+document.getElementById('email').value = parsedData.email;
+document.getElementById('textArea').value = parsedData.comment;
+function localStorageMine() {
+  const userName = document.getElementById('fullName');
+  const userEmail = document.getElementById('email');
+  const userComment = document.getElementById('textArea');
+  const uname = userName.value;
+  const email = userEmail.value;
+  const comment = userComment.value;
+  const inputFieldObject = {
+    name: uname,
+    email,
+    comment,
+  };
```

Comment:
- [ ] Great work implementing required changes from the previous reviewer. However, there's one more thing you guys need to work on before your project could be approved. It is required that when the When the user loads the page, the code checks if there's any saved data in the local storage and if so the input fields are pre-filled with the data. What this means is that, if the user enters data into the form without submitting the form, when the user reload the page, the user's data should remain on the form. Kindly check below code for insight :point_down: 

```
    window.onload = () => {
      let savedFormData = localStorage.getItem('contactFormInput');
      savedFormData = JSON.parse(savedFormData);
      // Check if the form data object is found on localStorage
      if (savedFormData) {
        // populate inputs values if data was found
        // ex: nameInput.value = savedFormData.name
       }
    };

```